web/libs/datamanager/src/sdk/lsf-sdk.js

```diff
@@ -917,12 +917,29 @@ export class LSFWrapper {
     const draftTime = Number(this.task.drafts[0]?.lead_time ?? 0);
     const lead_time = sessionTime + submittedTime + draftTime;
 
+    // StartedAt is the time in which the user first began labeling the task.
+    // It is the earliest time in the task.drafts array or the loadedDate if there are no drafts.
+    // When considering the draft created_at time, the difference between the draft lead_time is subtracted from the current time.
+    // This is done to get an adjusted startedAt time that accounts for the time the user was not actively labeling the task while in a draft state.
+    // This adjustment is necessary to ensure that the started_at time is not before the draftStartedAt time.
+    const draftStartedAt = draft ? new Date(this.task.drafts[0]?.created_at) : undefined;
+
+    let startedAt;
+    if (draftStartedAt) {
+      const draftLeadTime = Number(this.task.drafts[0]?.lead_time ?? 0);
+      const adjustedStartedAt = new Date(Date.now() - draftLeadTime * 1000);
+      startedAt = adjustedStartedAt < draftStartedAt ? draftStartedAt : adjustedStartedAt;
```

Comment:
this is a little ambiguous but i think intent is
```suggestion
      startedAt = adjustedStartedAt < (draftStartedAt ? draftStartedAt : adjustedStartedAt);
```

Reply:
or it could be 
```suggestion
      startedAt = (adjustedStartedAt < draftStartedAt) ? draftStartedAt : adjustedStartedAt;
```

Reply:
It is just a lower bound clamp is all it is. 

Either startedAt is the adjusted value, unless it is less than the time in which the draft was started.

Reply:
fair enough but reading it it could be interpreted in multiple ways which would cause confusion

Reply:
I applied the second change, as that is exactly the description I gave in the comment.